src/lava/proc/dense/models.py

```diff
@@ -49,42 +50,40 @@ class PyDenseModelBitAcc(PyLoihiProcessModel):
 
     s_in: PyInPort = LavaPyType(PyInPort.VEC_DENSE, bool, precision=1)
     a_out: PyOutPort = LavaPyType(PyOutPort.VEC_DENSE, np.int32, precision=16)
-    a_buff: np.ndarray = LavaPyType(np.ndarray,np.int32,precision=16)
+    a_buff: np.ndarray = LavaPyType(np.ndarray, np.int32, precision=16)
     weights: np.ndarray = LavaPyType(np.ndarray, np.int32, precision=8)
-    weight_exp: np.ndarray = LavaPyType(np.ndarray,np.int32, precision=4)
-    num_weight_bits: np.ndarray = LavaPyType(np.ndarray,np.int32,precision=3)
-    sign_mode: np.ndarray = LavaPyType(np.ndarray,np.int32,precision=2)
+    weight_exp: np.ndarray = LavaPyType(np.ndarray, np.int32, precision=4)
+    num_weight_bits: np.ndarray = LavaPyType(np.ndarray, np.int32, precision=3)
+    sign_mode: np.ndarray = LavaPyType(np.ndarray, np.int32, precision=2)
 
     def __init__(self):
-        super(PyDenseModelBitAcc,self).__init__()
-        #Flag to determine whether weights have already been scaled.
+        super(PyDenseModelBitAcc, self).__init__()
```

Comment:
Shorter: super().__init__(self)
There shouldn't be any confusion about which parent __init__ to pick.